ui/pages/confirmation/templates/add-ethereum-chain.js

```diff
@@ -134,11 +149,18 @@ const ERROR_CONNECTING_TO_RPC = {
 async function getAlerts(pendingApproval, state) {
   const alerts = [];
   let safeChainsList = [];
+  let providerError;
   if (state.useSafeChainsListValidation) {
-    safeChainsList = await fetchWithCache({
-      url: 'https://chainid.network/chains.json',
-      functionName: 'getSafeChainsList',
-    });
+    try {
+      safeChainsList = await fetchWithCache({
+        url: 'https://chainid.network/chains.json',
```

Comment:
Not required for a bug fix, but while we're here should we put this in a constant?

We could even split this logic and create a `validateChain` which does everything up to line 170 and returns `undefined` or the matched chain object.